lib/clipboard.rb

```diff
@@ -22,4 +22,12 @@ def self.insert!(html, plain_text, slack_text)
 
     pasteboard.put item
   end
+
+  def self.insert_branch!(branch_name)
```

Comment:
this method is simply inserting plain text onto the clipboard (versus html, slack_text, and plain_text), and could be re-used by other consumers. No need to couple it to 'branch_name'.

two routes:

1) did you try just using the existing `insert!` and providing either `nil` or the same value for the html and plain text args.. `ie Clipboard.insert!(branch_name, branch_name, nil)`

2) if that doesn't work for some reason, this method should just be named more generally, like:

```
def self.insert_text!(value)
  ...
end
```